.buildkite/pipeline-raw.yml

```diff
@@ -170,14 +170,14 @@ steps:
      build.branch == "master" &&
        ( build.message =~ /^Merge pull request .* from serokell\/auto\/v[0-9]+\.[0-9]+-release/ ||
          build.message =~ /^Merge pull request .* from serokell\/auto\/update-brew-formulae-v[.0-9]+-[0-9]+/ )
-   command: git push git@github.com:serokell/tezos-packaging-stable.git
+   command: git push --mirror git@github.com:serokell/tezos-packaging-stable.git
 
  - label: update RC mirror repository
```

Comment:
I have one additional small suggestion here: make these two steps depend on the `autorelease` one.

`git push --mirror` will push all reference, including tags, which means that if we wait for the `autorelease` step we'll also have new tags pushed in the mirrors too